src/components/ha-cards.html

```diff
@@ -88,6 +88,7 @@
     persistent_notification: 0,
     plant: 3,
     weather: 4,
+    custom_full_card: 0,
```

Comment:
Is there an agreement on the backend PR? If not - let's drop this line change for now

Reply:
You are right, I wanted to avoid merge conflicts in my first approach. Will submit a new PR